test/test_errors.py

```diff
@@ -20,19 +20,19 @@
 
 from pymongo.errors import (BulkWriteError,
                             EncryptionError,
-                            NotMasterError,
+                            NotPrimaryError,
                             OperationFailure)
 from test import (PyMongoTestCase,
                   unittest)
 
 
 class TestErrors(PyMongoTestCase):
```

Comment:
Please add a test that raises NotMasterError and excepts NotPrimaryError.

Reply:
Done.